client/extensions/woocommerce/app/settings/shipping/shipping-zone-list.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import ExtendedHeader from '../../../components/extended-header';
+import ShippingZone from './shipping-zone';
+import ShippingZoneDialog from './shipping-zone-dialog';
+
+class ShippingZoneList extends Component {
+	constructor( props ) {
+		super( props );
+
+		//TODO: use redux state with real data
+		this.state = {
+			showDialog: false,
+			shippingZones: [ {
+				locationName: 'United States',
+				locationDescription: '50 states',
+				methodName: 'USPS',
+				methodDescription: 'All domestic services',
+				icon: 'location'
+			}, {
+				locationName: 'Rest of the world',
+				locationDescription: '240 countries',
+				methodName: 'USPS',
+				methodDescription: 'All international services',
+				icon: 'globe'
+			} ],
+		};
+	}
+
+	renderShippingZone( zone, index ) {
+		return ( <ShippingZone key={ index } { ...zone } /> );
+	}
+
+	render() {
+		const { translate } = this.props;
+
+		const onAddZoneOpen = () => {
+			this.setState( { showDialog: true } );
+		};
+
+		const onAddZoneClose = () => {
+			this.setState( { showDialog: false } );
+		};
+
+		return (
+			<div>
+				<ExtendedHeader
+					label={ translate( 'Shipping Zones' ) }
+					description={ translate( 'The regions you ship to and the methods you will provide.' ) }>
+					<Button onClick={ onAddZoneOpen }>{ translate( 'Add zone' ) }</Button>
+				</ExtendedHeader>
+				<Card className="shipping__zones">
+					<div className="shipping__zones-row shipping__zones-header">
+						<div className="shipping__zones-row-icon"></div>
+						<div className="shipping__zones-row-location">{ translate( 'Location' ) }</div>
+						<div className="shipping__zones-row-method">{ translate( 'Shipping method' ) }</div>
```

Comment:
I think Shipping method should be Shipping methods since a zone can have more than one